modin/experimental/pandas/test/test_io_exp.py

```diff
@@ -182,7 +182,8 @@ def test_read_json_glob(filename, compression):
         df.to_json(filename, compression=compression)
 
     try:
-        df = pd.read_json_glob(filename, compression=compression)
+        json_glob_df = pd.read_json_glob(filename, compression=compression)
+        df_equals(json_glob_df, df)
```

Comment:
does `df_equals()` sort the things? if it doesn't, this is prune to sporadic failures because the order of rows could be different depending on the Moon phases...